src/appleseed.studio/mainwindow/mainwindow.cpp

```diff
@@ -124,6 +125,8 @@ MainWindow::MainWindow(QWidget* parent)
   , m_attribute_editor(0)
   , m_project_file_watcher(0)
 {
+    initialize_ocio();
```

Comment:
Genuine question: is this the right place to initialize OCIO? Shouldn't it be done at startup?